daemon/info_unix.go

```diff
@@ -184,6 +185,51 @@ func (daemon *Daemon) fillPlatformVersion(ctx context.Context, v *types.Version,
 		}
 		log.G(ctx).WithError(err).Warn("Failed to fill rootless version")
 	}
+
+	if err := daemon.fillModuleversion(ctx, v); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (daemon *Daemon) fillModuleversion(ctx context.Context, v *types.Version) error {
+	info, ok := debug.ReadBuildInfo()
```

Comment:
We are also reading build info in https://github.com/moby/moby/blob/f835ff6987c5f907eebbc4d709fd6400e41cfe8b/daemon/internal/builder-next/worker/mod/mod.go#L62. Wonder if we should share it?

Reply:
Yeah, I was also considering if some of these should be injected into the daemon when constructing it; i.e., if the `cmd/dockerd` package should construct version information, and passing it to the `daemon.New()` constructor. (I think we were discussing some of that related to https://github.com/moby/moby/pull/46464, and https://github.com/moby/moby/pull/50587, but I'm trying to find back that discussion 😓)